dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -443,7 +443,7 @@ const findLatestCompletedAmendmentByFacilityIdVersion = async (facilityId) => {
 };
 exports.findLatestCompletedAmendmentByFacilityIdVersion = findLatestCompletedAmendmentByFacilityIdVersion;
 
-exports.getLatestCompletedValueAmendment = async (req, res) => {
```

Comment:
I updated the names of a couple of functions here because it confused me and thought this was better - very trivial happy to undo if anyone wants me to